lib/hyrax/configuration.rb

```diff
@@ -381,14 +381,20 @@ def google_analytics_id
 
     attr_writer :permission_levels
     def permission_levels
-      @permission_levels ||= { "Choose Access" => "none",
-                               "View/Download" => "read",
-                               "Edit" => "edit" }
+      @permission_levels ||= { "View/Download" => "read",
```

Comment:
Would you also consider i18nizing the label strings in the methods you touched here too?

This may require undoing the memoization, to allow for translation to occur every time the method is called.